src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/FullBlockProcessor.scala

```diff
@@ -232,7 +232,7 @@ trait FullBlockProcessor extends HeadersProcessor {
     val toRemove: Seq[ModifierId] = heights.flatMap(h => headerIdsAtHeight(h))
       .flatMap(id => typedModifierById[Header](id))
       .flatMap(_.sectionIds.map(_._2))
-    historyStorage.remove(toRemove)
+    historyStorage.remove(Seq.empty, toRemove)
```

Comment:
Just a performance hint. The fastest way to obtain empty Seq is to use `mutable.WrappedArray.empty[T]`